data_tools/add.py

```diff
@@ -102,9 +99,9 @@
         added += 1
     
     print ("About to batch add: {}".format(added, skipped_existing))
-    fcw.set_messages_content_batch(DATASET_ID, messages_to_write)
+    fcw.set_dataset_messages_content_batch(DATASET_ID, messages_to_write)
```

Comment:
This seems like a superflous rename. Different PR please.

Reply:
We were trying to distinguish the function which applies to a whole dataset from the one that applies only to a single shard. If you don't think that's important, I can change back.